cs251tk/webapp/web_cli.py

```diff
@@ -0,0 +1,129 @@
+import os
+
+from ..common import chdir
+from ..student import stash, pull, checkout_date
+from ..student.markdownify.process_file import process_file
+from PyInquirer import style_from_dict, Token, prompt
+
+
+def check_student(student, spec, basedir):
+    files = []
+    if os.path.exists('{}/{}'.format(student, spec['assignment'])):
+        with (chdir('{}/{}'.format(student, spec['assignment']))):
+            for file in spec['files']:
+
+                result = process_file(file['filename'],
+                                      steps=file['commands'],
+                                      options=file['options'],
+                                      spec=spec,
+                                      cwd=os.getcwd(),
+                                      supporting_dir=os.path.join(basedir, 'data', 'supporting'),
+                                      interact=False,
+                                      basedir=basedir,
+                                      spec_id=spec['assignment'],
+                                      skip_web_compile=False)
+
+                if 'web' in file['options']:
+                    if result['missing']:
+                        if 'optional' in file['options']:
+                            files = files + [file['filename'] + ' MISSING (OPTIONAL)']
+                        else:
+                            files = files + [file['filename'] + ' MISSING']
+                    else:
+                        files = files + [file['filename']]
+                else:
+                    continue
+    return files
+
+
+def ask_student(usernames):
+    style = style_from_dict({
+        Token.QuestionMark: '#959ee7 bold',
+        Token.Selected: '#959ee7',
+        Token.Pointer: '#959ee7 bold',
+        Token.Answer: '#959ee7 bold',
+    })
+    questions = [
+        {
+            'type': 'list',
+            'name': 'student',
+            'message': 'Choose student',
+            'choices': ['QUIT', 'LOG and QUIT'] + usernames
```

Comment:
```suggestion
            'choices': ['QUIT', 'LOG and QUIT', *usernames]
```

using the splat operator to spread a list into another feeeeeeels nicer to me here?